app/assets/stylesheets/card.scss

```diff
@@ -38,12 +38,16 @@ body {
 
   a {
     color: $teal;
-    margin-right: 1.3rem;
+    // margin-right: 1.3rem;
     text-decoration: none;
     text-transform: uppercase;
     transition: color 0.3s ease;
   }
 
+  a:hover {
+    color: #259cbd;
```

Comment:
Color literals like `#259cbd` should only be used in variable declarations; they should be referred to via variable everywhere else.